Emma/emma_libs/memoryManager.py

```diff
@@ -208,87 +198,116 @@ def createStandardReports():
         #     print(graph.source)
         def createSvgReport(startPoint, endPoint):
             """
-            plot sections and objects of a given memory area
+            Plot sections and objects of a given memory area
             :param startPoint: beginning of address area
             :param endPoint: end of address area
             """
-
-            def draw(image, elementstoPlot, startPoint, y, colour):
+            class Element(IntEnum):
+                addressStart = 0
+                addressEnd = 1
+                addressLength = 2
+                fqn = 3
+                originalAddressStart = 4
+
+            class PlottedElement(IntEnum):
+                addressEnd = 0
+                yAxe = 1
+
+            def drawElements(image, elementsToPlot, startPoint, y, colour):
+                """
+                :param image: svgwrite.Drawing object
+                :param elementsToPlot: [[addressStart, addressEnd, addressLength, fqn, originalAddressStart], ...] List of objects to plot (-> accessed via Element enum)
+                :param startPoint: Beginning of the address area
+                :param y: Start point on the y axis
+                :param colour: [str] Text colour (see HTML reference)
+                :return: None
+                """
                 yAxe = y
-                lastSectionPosition = 5
+                lastSectionPosition = 5             # Determine the deepest (with highest y value) section element, used to plot objects so they do not cross section elements; in pixel
                 biggestEndAddress = 0
                 plottedElements = []
                 distanceBetweenElements = 15
```

Comment:
This is no distance but an drawing offset
```suggestion
                drawingOffset= 15
```